frappe/__init__.py

```diff
@@ -2231,7 +2231,7 @@ def get_website_settings(key):
 	if not hasattr(local, "website_settings"):
 		local.website_settings = db.get_singles_dict("Website Settings", cast=True)
 
-	return local.website_settings[key]
+	return local.website_settings.get(key)
```

Comment:
Similar issue waiting to surface just 3 line below 👀  

https://github.com/frappe/frappe/blob/d7e51d628ab2d11960520be5dc00b7e49db9ba5c/frappe/__init__.py#L2241